returnn/datasets/audio.py

```diff
@@ -469,7 +469,7 @@ def get_corpus_seq(self, corpus_seq_idx: int) -> DatasetSeq:
             with self._open_audio_file(corpus_seq_idx) as audio_file:
                 features = self.feature_extractor.get_audio_features_from_raw_bytes(audio_file, seq_name=seq_tag)
         else:
-            features = numpy.zeros((), dtype=numpy.float32)  # currently the API requires some dummy values...
+            features = numpy.zeros((0, 0), dtype=numpy.float32)  # currently the API requires some dummy values...
```

Comment:
Why do you need this? You should not care at all about keys which are not defined by the dataset (not in `get_data_keys()`).

Reply:
The issue is that the OggZipDataset does in fact output `data` in `get_data_keys()` for historical reasons, I think.

Reply:
Ah that's an error then in `OggZipDataset`.

Reply:
So, as said, this change should not be done here. You should fix `get_data_keys` instead.

Reply:
Why is there that comment then though? It seems that it is somehow required in other parts of the code that there is a data entry?

Reply:
This comment is about the API of `DatasetSeq`.

Actually, I think it's a bit outdated. You don't need to pass `targets` anymore, you can directly pass `features` as a dict, and then can also omit `"data"` completely. But otherwise, `features` must be some `ndarray`, but otherwise it does not matter.